rfcs/top-bar-silhouette.md

```diff
@@ -0,0 +1,62 @@
+# Top Bar Silhouette
+
+This silhouette provides proper styles for wrapper element to be used as Top Navigation. By default element has no visual styles.
+Silhouettes comply with Base UI grid layout, so they are responsive to resizing and stretching, changing sizes, fonts, etc
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {TopBar} from 'baseui/silhouette/top-bar';
+import {Button} from 'baseui/button';
+
+export default () => (
+  <TopBar>
+    <Button>Menu</Button>
+    <span>Uber</span>
+  </TopBar>
+  );
+```
+
+### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {TopBar} from 'baseui/silhouette/top-bar';
```

Comment:
as I see, this naming is misleading - we want to provide layouts, and not just topbars, but a Layout that has a topbar and a body.

What would you think about this approach?

```
<Layout type={Layout.TOP_BAR} placements={{
  header: HeaderNavigation,
  body: <div></div>
}}/>
```